docs/pages/router/reference/link-preview.mdx

```diff
@@ -0,0 +1,228 @@
+
+<ContentSpotlight
+  alt="A video presenting the link preview feature in Expo Router."
+  file="/expo-router/link-preview.mp4"
+/>
+
+This guide will show you how to add and customize a link preview for your app on iOS. A link preview is a feature commonly used on iOS to show user a preview of the content linked by the link.
+If you have a link in your app, you can add preview by simply wrapping the link content with `<Link.Trigger>` component and adding `<Link.Preview>` component. This will create a preview of the page that the link points to.
+
+```tsx
+import { Link } from 'expo-router';
+
+export default function Page() {
+  return (
+    <Link href="/about">
+      <Link.Trigger>About</Link.Trigger>
+      <Link.Preview />
+    </Link>
+  );
+}
+```
+
+## New components
+
+### Link.Trigger
+
+A wrapper for the link's trigger. Anything that was previously passed as a child to `<Link>` should now be wrapped inside `<Link.Trigger>` when using `<Link.Preview>`.
+
+### Link.Preview
+
+Enables the link preview. You can customize its behavior and appearance. For more information see the section below on customizing previews.
+
+```tsx
+export default function Page() {
+  return (
+    <Link href="/about">
+      <Link.Trigger>About</Link.Trigger>
+      <Link.Preview />
+    </Link>
+  );
+}
+```
+
+<ContentSpotlight
+  alt="A screenshot of an iOS link preview."
+  src="/static/images/expo-router/link-preview-preview.png"
+  className="max-h-[400px]"
+/>
+
+### Link.Menu & Link.MenuAction
+
+Enables the context menu. Use `<Link.Menu>` as a container and `<Link.MenuAction>` for individual menu items.
+
+```tsx
+<Link href="/about">
+  <Link.Trigger>About</Link.Trigger>
+  <Link.Menu>
+    <Link.MenuAction title="Share" icon="square.and.arrow.up" onPress={handleSharePress} />
+    <Link.MenuAction title="Block" icon="nosign" destructive onPress={handleBlockPress} />
+  </Link.Menu>
+</Link>
+```
+
+<ContentSpotlight
+  alt="A screenshot of an iOS context menu."
+  src="/static/images/expo-router/link-preview-menu.png"
+  className="max-h-[400px]"
+/>
+
+## Customizing the preview
+
+By default, the preview is rendered as a full-sized page snapshot. There are several ways to customize this behavior.
+
+### Custom size
+
+You can use the `width` and `height` props to suggest a preferred preview size. The system will consider these preferences but may override them based on available space or platform behavior.
+
+```tsx
+<Link href="...">
+  <Link.Trigger>Content</Link.Trigger>
+  <Link.Preview height="" />
+</Link>
+```
+
+<ContentSpotlight
+  alt="A screenshot of a custom link preview size."
+  src="/static/images/expo-router/link-preview-custom-size.png"
+  className="max-h-[400px]"
+/>
+
+### Custom preview
+
+If you don't want to show the default preview, you can pass custom content to the `<Link.Preview>` component via children. This custom content will replace the default preview of the link target.
```

Comment:
```suggestion
If you don't want to show the default preview, you can pass custom content to the `Link.Preview` component via children. This custom content will replace the default preview of the link target.
```

We reserver `<ComponentName>` pattern for common React Native components. So let's avoid `<>` for our own.